vignettes/AssuranceConsecutiveStudies.Rmd

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Bayesian Assurance Using East-R"
+title: "Bayesian Assurance of Consecutive Studies"
```

Comment:
@kwathen, can you please confirm if this renaming is correct?

Reply:
Just to clarify: I tried to make consistent the titles over all pages of CyneRgy's website, but I did not invent any new name. The name "Bayesian Assurance of Consecutive Studies" was from the Examples list. It seemed like the best to keep between the two.

Reply:
It should be Bayesian Assurance of Consecutive Studies